assembl/static2/tests/unit/components/form/multilingualRichTextFieldAdapter.spec.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import DummyForm from './dummyForm';
+import { createEditorStateFromText } from '../../../helpers/draftjs';
+import MultilingualRichTextFieldAdapter from '../../../../js/app/components/form/multilingualRichTextFieldAdapter';
+
+// avoid random stuff in editor state and editor component to be able to use snapshots
+jest.mock('draft-js/lib/generateRandomKey', () => () => '123');
+jest.mock('../../../../js/app/components/common/richTextEditor/index', () => 'AssemblEditor');
+
+describe('MultilingualRichTextFieldAdapter component', () => {
+  const onChangeSpy = jest.fn();
+  const onFocusSpy = jest.fn();
+
+  it('should render a rich text field', () => {
+    const props = {
+      editLocale: 'fr',
+      input: {
+        name: 'text',
+        onChange: onChangeSpy,
+        onFocus: onFocusSpy,
+        value: {
+          en: createEditorStateFromText('Hello'),
+          fr: createEditorStateFromText('Bonjour')
+        }
+      },
+      label: 'Text',
+      meta: {
+        error: '',
+        touched: false
+      }
+    };
+    const component = renderer.create(
+      <DummyForm>
+        <MultilingualRichTextFieldAdapter {...props} />
+      </DummyForm>
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
Hey @cedricmessiant what's proving you that the rich text field is rendered here ? Saying that it simply matching a snapshot doesn't mean it appears. And also you have to manually check the snapshot created to make sure that the component is there. Same comment for the other `toMatchSnapshot` tests

Reply:
I don't understand. I checked the snapshot when it has been created the first time and then everytime a test complains that it dosen't match the snapshot anymore, I figure out what happens to determine if something is wrong or if the snapshot needs to be updated.